cmd/containerd/command/main.go

```diff
@@ -130,6 +132,19 @@ can be used and modified as necessary as a custom configuration.`
 			return err
 		}
 
+		// Initialize openTelemetry tracing
+		shutdown, err := tracing.InitOpenTelemetry(config)
+		log.G(ctx).Debug("Initializing opentelemetry")
```

Comment:
- "Initializing ..." message should be logged before calling InitOpenTelemetry
- In logging and error messages, we should use "OpenTelemetry" instead of "opentelemetry", "openTelemetry" and/or "open telemetry".

Reply:
Removing the initializing message anyway.

Reply:
Removed the initialization message.